mozdownload/scraper.py

```diff
@@ -77,6 +77,18 @@
 }
 
 
+def thunderbird_latest_version_filter(x):
+    invalid_versions = ("0.7rc", "1.0rc", "2.0.0.0", "2.0.0.0rc1", "10.0-real")
```

Comment:
I would propose to rename the `invalid_versions` variable given that those releases exist but simply don't have a valid version number.

```suggestion
    accepted_invalid_versions = ("0.7rc", "1.0rc", "2.0.0.0", "2.0.0.0rc1", "10.0-real")
```

It would be great if you could also add a doc comment explaining why this is needed.